packages/common/src/ui/components/Banner/Banner.tsx

```diff
@@ -53,11 +68,22 @@ export const Banner = ({
         </div>
       )}
       <div
-        className={cn(styles.descriptionContainer, { [descriptionClassName]: descriptionClassName })}
+        className={cn(styles.contentContainer, { [descriptionClassName]: descriptionClassName })}
         data-testid="banner-description"
       >
-        <Text className={styles.message}>{message}</Text>
-        {description && <div>{descriptionElement}</div>}
+        <div className={cn(styles.descriptionContainer)}>
+          <Text className={styles.message}>{message}</Text>
+          {withLink && (
+            <>
+              <Link to="">{linkMessage}</Link>
+              <Text className={styles.message}>{messagePartTwo}</Text>
+            </>
+          )}
+          {description && <div>{descriptionElement}</div>}
+        </div>
+        <div className={cn(styles.buttonContainer)}>
+          {withButton && <Button onClick={onButtonClick}> {buttonMessage} </Button>}
```

Comment:
`withButton` is an unnecessary prop since it does not do anything else in this component apart from this.

```suggestion
          {buttonMessage && <Button onClick={onButtonClick}> {buttonMessage} </Button>}
```